### DISCUSSION TITLE
trajectory analysis

### DISCUSSION THREAD

**Discussion by rojinsafavi** [GreenleafLab/ArchR#775] [Questions / Documentation]
I was wondering if it's possible to get a trajectory similar to Signac implementation in ArchR? I very much like to see different branching points in my dataset

![image](https://user-images.githubusercontent.com/19545198/119734477-ecc4e880-be2f-11eb-93e5-41358489df2d.png)

**Comment by rcorces** [comment]
Have you tried ArchR's `addMonocleTrajectory()` function? https://www.archrproject.com/reference/addMonocleTrajectory.html

**Reply by rojinsafavi** [reply]
Didn't see that! perfect, will try that, thank you!

**Comment by NoemieL** [comment]
Hi. I imagine that with monocle and Slingshot trajectory analysis, you are using the scATACseq, but could you please confirmed? (I did not find which data you were using in the code) Also do you think it will be able to add an option to use the GeneIntegrationScore? I am curious to see if we get the same trajectory prediction.
Thanks

**Reply by rcorces** [reply]
yes, all trajectories on ATAC. we dont have plans to add support for gene integration score yet but seems like a reasonable idea. Feel free to open a separate discussion topic under feature requests if you'd like us to consider it in the future.

**Reply by NoemieL** [reply]
Thanks for the quick answer!
